src/main/java/io/supertokens/multitenancy/MultitenancyHelper.java

```diff
@@ -111,11 +111,11 @@ public List<TenantIdentifier> refreshTenantsInCoreBasedOnChangesInCoreConfigOrIf
 
                     Map<ResourceDistributor.KeyClass, JsonObject> normalizedTenantsFromDb =
                             Config.getNormalisedConfigsForAllTenants(
-                            tenantsFromDb, Config.getBaseConfigAsJsonObject(main));
+                            this.getFilteredTenantConfigs(tenantsFromDb), Config.getBaseConfigAsJsonObject(main));
```

Comment:
add a new variable instead of using this function everywhere

Reply:
Done